src/components/Login/Login.js

```diff
@@ -0,0 +1,92 @@
+import "./Login.scss";
+
+import Navbar from "../Navbar/Navbar";
+import LandingVideo from "../LandingVideo/LandingVideo";
+import FormInput from "../FormInput/FormInput";
+import UserService from "../../services/userService";
+import { useState } from "react";
+
+import smiles from "../../assets/smilies.svg";
```

Comment:
```suggestion
import { useState } from "react";

import smiles from "../../assets/smilies.svg";
import UserService from "../../services/userService";
import Navbar from "../Navbar/Navbar";
import LandingVideo from "../LandingVideo/LandingVideo";
import FormInput from "../FormInput/FormInput";
import "./Login.scss";
```
Also, you could use absolute imports, instead of relative ones. You can check something like here: https://plainenglish.io/blog/why-and-how-to-use-absolute-imports-in-react

Reply:
Nice suggestion!